src/main/java/com/ft/methodearticleinternalcomponentsmapper/MethodeArticleInternalComponentsMapperApplication.java

```diff
@@ -116,6 +116,7 @@ public void run(final MethodeArticleInternalComponentsMapperConfiguration config
         Map<String, MethodeArticleValidator> articleValidators = new HashMap<>();
         articleValidators.put(InternalComponentsMapper.SourceCode.FT, new MethodeArticleValidator(mamClient, mamUri, mamConfiguration.getHostHeader()));
         articleValidators.put(InternalComponentsMapper.SourceCode.CONTENT_PLACEHOLDER, new MethodeArticleValidator(mcpmClient, mcpmUri, mcpmConfiguration.getHostHeader()));
+        articleValidators.put(InternalComponentsMapper.SourceCode.DynamicContent, new MethodeArticleValidator(mamClient, mamUri, mamConfiguration.getHostHeader()));
```

Comment:
If `InternalComponentsMapper.SourceCode.DynamicContent` is a constant, then it should probably be uppercase.